.vscode/settings.json

```diff
@@ -5,6 +5,5 @@
 	},
 	"search.exclude": {
 		"out": true // set this to false to include "out" folder in search results
-	},
-	"typescript.tsdk": "./node_modules/typescript/lib" // we want to use the TS server from our node_modules folder to control its version
+	}
 }
```

Comment:
Hey, this is overwritten by autogenerated file from latest yeoman.
I just added it back. = )